pkgs/tools/video/svt-av1/default.nix

```diff
@@ -1,20 +1,35 @@
-{ lib, stdenv, fetchFromGitLab, cmake, nasm }:
+{ lib
+, stdenv
+, fetchFromGitLab
+, gitUpdater
+, cmake
+, nasm
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "svt-av1";
-  version = "1.2.1";
+  version = "1.4.1";
 
   src = fetchFromGitLab {
     owner = "AOMediaCodec";
     repo = "SVT-AV1";
-    rev = "v${version}";
-    sha256 = "sha256-gK2Yabh9AwAX1AecOPGTOthE4ENCA4NIjwWNJNkXxJc=";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-jmr5egbuqLnBW7OFuaQk3F4s5xqTpXhWcJAfZySGWeU=";
   };
 
-  nativeBuildInputs = [ cmake nasm ];
+  nativeBuildInputs = [
+    cmake
+    nasm
+  ];
+
+  passthru.updateScript = gitUpdater {
+    rev-prefix = "v";
+  };
 
   meta = with lib; {
+    homepage = "https://gitlab.com/AOMediaCodec/SVT-AV1";
     description = "AV1-compliant encoder/decoder library core";
+
```

Comment:
There should not be blank lines here

Reply:
Is there a standard for this in nixpkgs?
I personally think a (newline) separation between single and multi-line declarations is more readable.

Reply:
> Is there a standard for this in nixpkgs?

there isn't. there isn't even a recommendation. whatever anyone comments is their personal preference.

there's https://github.com/NixOS/rfcs/pull/101, but that's still not merged.